mibs/ericsson/PT-ALL-MIB

```diff
@@ -0,0 +1,4919 @@
+MINI-LINK-MIB DEFINITIONS ::= BEGIN
```

Comment:
This file is in fact a "merge" of all the MIB files. It can (and should in fact) be deleted so we only keep the separate MIBs. Please check that all you standalone files are correctly added.